include/zephyr/net/dhcpv4.h

```diff
@@ -44,6 +98,44 @@ enum net_dhcpv4_state {
 
 /** @endcond */
 
+/**
+ * @brief Helper to initialize a struct gpio_callback properly
```

Comment:
It's `struct net_dhcpv4_option_callback`

Reply:
Sorry, will fix it